Assets/Scripts/MainMenu/SettingsScripts/ApplyScript.cs

```diff
@@ -2,67 +2,67 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using System;
 
-namespace MainMenu
```

Comment:
it won't detect the PlayerData class that's outside of the namespace. Not sure if it would still work after we delete PlayerData within the namespace, but we can try

Reply:
> it won't detect the PlayerData class that's outside of the namespace
Why not use `using <namespace>`?

Reply:
Of course [PlayerData](https://github.com/hunsri/EwMM-ProjectCS/blob/ft/main-menu-data/Assets/Scripts/Datas/PlayerData.cs) would need to be updated to have a namespace first (e.g. Data)
We should consider adding a namespace to every script anyway

Reply:
> > it won't detect the PlayerData class that's outside of the namespace
> 
> Why not use `using <namespace>`?

Yes. I think this is actually a good idea and will add this tomorrow. But I still think we need to delete the `PlayerData` within the `MainMenu` namespace, as the `ApplyScript` would detect `PlayerData` as the one within its namespace first, right?

Reply:
I think [PlayerData](https://github.com/hunsri/EwMM-ProjectCS/blob/ft/main-menu-data/Assets/Scripts/MainMenu/SettingsScripts/PlayerData.cs) within the main menu should be refactored to a different name anyway.

Maybe to 'SettingsData', since it holds mainly settings anyway.

Reply:
I meant I moved all of its attributes to `PlayerData` class as it would make saving / loading much easier. Essentially it could be deleted for now as there are no script that's using it.